spec/features/Welcome/index_spec.rb

```diff
@@ -10,4 +10,39 @@
 
     expect(current_path).to eq("/quotes")
   end
+
+  describe "I am taken to '/quotes' where I see the TOTAL number of quotes that were found for that keyword" do
+    describe "As well as the keyword that's being used for the search" do
+      it "As well as a list of the top 10 quotes that were found" do
+        visit root_path
+
+        fill_in :keyword, with: 'Love'
+
+        click_on "Search for Quotes"
+
+        expect(current_path).to eq("/quotes")
+
+        expect(page).to have_content('Quotes Matching Search For:')
+
+        expect(page).to have_content('Iris Murdoch')
+        expect(page).to have_content('There is only one happiness in life, to love and be loved.')
+        expect(page).to have_content('Tags:')
```

Comment:
I am not sure if you updated this line or not but the one that I have is failing because it looks like this:
```
expect(page).to have_content('tags:')
```
Remember in these feature tests, you are essentially testing exactly what gets displayed on the page. So yes, `Tags:` does exist but I would also add what follows that. 
```
expect(page).to have_content('Tags: famous-quotes')
```
I also think that in many cases there was more than a single tag. I noticed that you called `.first` in your view but we would want all of the tags to display. So how could you take an array and make it a string separated by commas? And where might you want to do that?